dev-docs/RFCs/v7.x/post-process-effect-rfc.md

```diff
@@ -0,0 +1,66 @@
+# RFC: Post-processing Effect Rendering
+
+* **Authors**: Jian Huang
+* **Date**: May. 2019
+* **Status**: For Review
+
+## Abstract
+Post-processing effect rendering is part of deck.gl effect rendering pipeline, focusing on screen space pixel manipulation by using ping-pong technique.
+
+## Design
+
+A post-processing effect is created from `PostProcessEffect` class and Shader module.
+```js
+import {brightnessContrast} from @luma.gl/effects;
+import {PostProcessEffect} from @deck.gl/core;
+
+const postProcessEffect = new PostProcessEffect(brightnessContrast, {
+  brightness: 0.5,
+  contrast: 0.5
+});
+```
+
+### API
+
+`PostProcessEffect` class is the public interface to create post-processing effects, it only has two params.
+* `module`(Object): GLSL shader code and default uniforms for rendering
+* `props`(Object): parameters to tune effects
+
+`ScreenPass` class is the internal interface to handle the rendering of one pass, it is created by PostProcessEffect dynamically from shader module.
+* `inputBuffer`(Object): frame buffer object as input data
```

Comment:
`inputBuffer` and `outputBuffer` naming implies these are `Buffer` objects, suggest rename these, may be `inputFrame` or `inputFramebuffer` ?